apps/event-service/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask==2.2.2
 supabase==0.7.1
+datetime==5.0
```

Comment:
`datetime` is a native Python module; why do you need to install this for the app to run?